<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="label_login">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 102); font: 87 28pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>ADD PAYMENT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="foodbutton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
background-color: rgb(54, 54, 54);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>../images/food.ico</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>101</width>
     <height>101</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_choosed">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,0,102); font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>670</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt;color:rgb(54,54,54);
background-color: rgb(255,0,102);
font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_price">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>530</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cleaningbutton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
background-color: rgb(54, 54, 54);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>../images/cleaning.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>101</width>
     <height>101</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="billbutton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
background-color: rgb(54, 54, 54);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>../images/bill.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>101</width>
     <height>101</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="transportbutton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
background-color: rgb(54, 54, 54);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>../images/vehicles.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>101</width>
     <height>101</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="selfcarebutton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
background-color: rgb(54, 54, 54);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>../images/personal-care.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>101</width>
     <height>101</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="rentbutton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
background-color: rgb(54, 54, 54);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>../images/house.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>101</width>
     <height>101</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="othersbutton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
background-color: rgb(54, 54, 54);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>../images/others.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>101</width>
     <height>101</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="healthbutton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
background-color: rgb(54, 54, 54);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>../images/healthcare.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>101</width>
     <height>101</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="snackbutton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
background-color: rgb(54, 54, 54);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>../images/snack.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>101</width>
     <height>101</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_price">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,0,102);
font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_price_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>575</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,0,102);
font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Comment:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_comment">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>575</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>670</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt;color:rgb(54,54,54);
background-color: rgb(255,0,102);
font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_warning">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>740</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255); font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>620</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 87 14pt &quot;Arial Black&quot;;

dateEdit_tabDetail-&gt;setDate(QDate::currentDate());</string>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2022</year>
     <month>10</month>
     <day>20</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2022</year>
     <month>10</month>
     <day>20</day>
    </datetime>
   </property>
   <property name="maximumDate">
    <date>
     <year>2030</year>
     <month>12</month>
     <day>31</day>
    </date>
   </property>
   <property name="minimumDate">
    <date>
     <year>2022</year>
     <month>10</month>
     <day>20</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,0,102);
font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Date:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
